tools/accuracy_checker/accuracy_checker/annotation_converters/super_resolution_converter.py

```diff
@@ -84,7 +100,11 @@ def configure(self):
     def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
         content_errors = [] if check_content else None
         file_list_lr = []
-        for file_in_dir in self.data_dir.iterdir():
+        lr_dir = self.lr_dir if self.lr_dir else self.data_dir
```

Comment:
I think directories specification can be moved in configure() method.
and what happens if both provided?